gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/test/java/io/gravitee/gateway/handlers/api/manager/ApiManagerTest.java

```diff
@@ -119,7 +119,7 @@ public void shouldDeployApiWithPlan() throws Exception {
 
     @Test(expected = IllegalArgumentException.class)
     public void shouldNotDeployApi_invalidTag() throws Exception {
-        shouldDeployApiWithTags("test,!test", new String[] {});
+        shouldDeployApiWithTags("test,!test", new String[] { "test" });
```

Comment:
I don't get the significance of this test 🧐
Actually, I'm not even sure of what should be the behavior of the gateway when it is configured with such `test,!test` tags.
If I just try to interpret it quickly, I would interpret like this:

> The Gateway deploys all api having the tag `test` but will not deploy the apis having the tag `test`

Will the API be really deployed?

Reply:
no the api won't be deployed as an error is raised if exclude/include stuff are wrongly set.